scripts/bundleConfig.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const config = require('../shared/data/constants.js')
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args))
+
+const outputEtagFile = './shared/data/etags.json'
+const bundlePath = './shared/data/bundled/'
+const configProps = config.tdsLists.find((c) => c.name == 'config');
+const etags = {};
+
+async function getConfig(name, url) {
+    let response = await fetch(url)
+    if (!response.ok) {
+        throw new Error(`Failed to load tds list ${name}: ${url}`)
+    }
+    etags[`${name}-etag`] = response.headers.get('etag')
+    const fileText = await response.text()
+    const urlObject = new URL(url)
+    const fileName = urlObject.pathname.split('/').pop()
+    fs.writeFileSync(`${bundlePath}/${fileName}`, fileText)
+}
+
+async function fetchConfigs() {
+    await getConfig('config', configProps.url)
+    fs.writeFileSync(outputEtagFile, JSON.stringify(etags))
```

Comment:
What about logging a success message here?

```suggestion
    fs.writeFileSync(outputEtagFile, JSON.stringify(etags))
    console.log('Config imported successfully')
```

Reply:
Good catch Emanuele - something is clearly wrong with linter setup though. Can you double check if that file is linted @jonathanKingston ?

Reply:
scripts/ are explicitly not linted because most of them are very broken lint-wise. I'll add this for now to the lint step and fix it up. I don't want to muddy this PR with lots of syntax changes... I'll make that a follow up.